pkg/server/file.go

```diff
@@ -115,23 +115,23 @@ func (t *TranslateServiceServer) UploadTranslationFile(
 	}
 
 	// Case for when not original, or uploading original for the first time.
-	updatedMessages := model.MessagesSlice{*messages}
+	updatedMessages := model.TranslationSlice{*messages}
 
-	// When updating original messages, changes might affect translations - transform and update all translations.
+	// When updating original translation, changes might affect translations - transform and update all translations.
 	if messages.Original {
-		all, err := t.repo.LoadMessages(ctx, params.serviceID, repo.LoadMessagesOpts{})
+		all, err := t.repo.LoadTranslation(ctx, params.serviceID, repo.LoadTranslationOpts{})
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "")
 		}
 
 		if origIdx := all.OriginalIndex(); origIdx != -1 {
 			oldOriginal := all[origIdx]
 
-			// Mark new or altered messages as untranslated.
+			// Mark new or altered translation as untranslated.
```

Comment:
same, as on update request.

// Compare repo and request original translation. Change status for new or altered translation.messages to UNTRANSLATED for all languages